modules/byo-vpc-multiple-az.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * assemblies/byo-vpc.adoc
+
+[id="byo-vpc-multiple-az_{context}"]
+= BYO VPC with multiple availability zones required fields
+
+. Create six subnets for three separate zoens as follows:
++
+|===
```

Comment:
usually we add attributes above the table. Even if you don't need to change the column widths, it's good because it should make that first row bold as it's the header

```
[options="header"]
```

Same for other tables in the PR if you can check